### PR TITLE
Fixes warning about activeStartDate required prop

### PR DESCRIPTION
**Pull request by same31**
I realised I introduced a warning by making the prop as required, my bad.

Also, do you think you can publish a new version of the package after this merge?
Thank you!

### CODE REVIEW

**Comment by wojtekmaj** [comment]
I don't think that's a valid fix actually. `activeStartDate` should be always available in tileDisabled, so you should be removing 

```js
    const {
      activeStartDate,
      ...otherProps
    } = this.props;
```

bit in `Decades.jsx`, `Years.jsx`, `Months.jsx`. Otherwise only month view will benefit from the change.

File: src/shared/propTypes.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const tileGroupProps = {
 };
 
 export const tileProps = {
-  activeStartDate: PropTypes.instanceOf(Date).isRequired,
```
Comment (wojtekmaj) [inline src/shared/propTypes.js]: Please keep it as `isRequired` - `tileDisabled` should always get this parameter, so it *is* required.

**Comment by same31** [comment]
Thank you for your review, that makes perfect sense!

Here are the requested changes.

**Comment by wojtekmaj** [comment]
Thank you!

**Pull request merged**